x-pack/legacy/plugins/maps/public/components/_index.scss

```
@@ -2,6 +2,26 @@
   margin-bottom: $euiSizeS;
 }
 
+.mapMetricEditorPanel__metricEditor {
+  padding: $euiSizeM 0;
+  border-top: 1px solid $euiColorLightShade;
```

Mostly FYI, we do have an overall variable for this which is 
```suggestion
  border-top:  $euiBorderThin;
```